torchbenchmark/models/speech_transformer/config.py

```diff
@@ -53,29 +53,30 @@ class SpeechTransformerTrainConfig:
     visdom_lr = 0
     visdom_epoch = 0
     visdom_id = 0
+    cross_valid = False
     # The input files. Their paths are relative to the directory of __file__
     train_json = "input_data/train/data.json"
     valid_json = "input_data/dev/data.json"
     dict_txt = "input_data/lang_1char/train_chars.txt"
-    def __init__(self):
+    def __init__(self, prefetch=True, train_bs=16, num_train_batch=1):
```

Comment:
Why is train_bs defaulted to 16 here? Also, I think it should set self.batch_size, otherwise that won't be used.

Reply:
This is the old value I set. Should remove the default value. Fixed.